user/pages/03.commerce2/03.user-guide/03.products/06.delete-a-product/docs.md

```diff
@@ -6,7 +6,13 @@ taxonomy:
 
 Sometimes you may wish to delete a product or one of its variations. Before deleting a product or variation, consider unpublishing the product or disabling the variation.
 
-! @todo the unpublish process is just a checkbox, review. No way to disable variations by default.
+## Disabling a product variation
+
+This will remove the product variation from the site, but will leave it in the system should you want to re-enable it at a later point.

```

Comment:
> This will remove the product variation from the site

Maybe change to "prevent the product variation from being displayed or purchased"?